pipeline.wdl

```diff
@@ -18,7 +18,8 @@ workflow pipeline {
         String outputDir
         Reference reference
         IndexedVcfFile? dbsnp
-        String starIndexDir
+        String? starIndexDir
```

Comment:
Shouldn't this be a file technically?
Cromwell should link this anyway, right? So no filesize issues if everything is working correctly.

Reply:
Yes, but I wasn't sure if changing this might cause any issues. Therefore, I felt this might be better to leave to a sperate PR.

Reply:
It will require a change in the STAR task as well.

Reply:
I already have made this change in the star task because it had issues when testing with relative paths https://github.com/biowdl/tasks/blob/78ebc0c8f54cb37c7ff3742725f7ccf1de96f3e0/star.wdl#L7

Plus, WDL parses strings as paths when strings are passed to a file variable and vice versa. So there is no need to worry about this.